### ISSUE TITLE
adjustRtime method for XChromatograms objects 

### ISSUE THREAD

**Issue by stanstrup** [sneumann/xcms#339]
With grouping it would be nice also to have retention time adjustment.

grouping: https://github.com/sneumann/xcms/issues/336

**Comment by jotsetung** [comment]
We just have to figure out also which methods might make most sense. *peak groups* might work, but eventually we need to implement a simple warping approach.

**Comment by stanstrup** [comment]
Yeah I was thinking "just" the density thing. I guess this might be a case where obiwarp would be preferable though.

I was considering looking into https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0198311 also. That could be used to first align and the you'd just need the XCMS grouping.

**Comment by jotsetung** [comment]
Here we can use this package https://cran.r-project.org/web/packages/dtw/index.html

**Comment by stanstrup** [comment]
Or https://cran.r-project.org/package=ptw
No idea what works best. Ptw seems to be the reference in many papers.
